### ISSUE TITLE
EBAS obs outlier removal?

### ISSUE THREAD

**Issue by jgliss** [metno/emep_trends_2021#3]
Default obs reading will use EBAS level 2 and invalidate invalid measurements based on flags in the NASA Ames files. Is there any more sanity checking to be done?

**Comment by jgliss** [comment]
E.g. outlier removal, certain sites that should not be used?

**Comment by SverreSolberg** [comment]
Hm. I just wonder if all data actually have a specified level. Can you check if there are time series without?

**Comment by jgliss** [comment]
If I remember correctly, I think we only receive level 2 data from NILU, but I can check for O3!

**Comment by jgliss** [comment]
@SverreSolberg  Apparently, it seems I am wrong here, the O3 data has a lot of files without a data level specified.

**Comment by jgliss** [comment]
should I remove the data_level=2 filter for O3?

**Comment by SverreSolberg** [comment]
Personally, I'm so frustrated about these datalevels myself, but dont tell that further ;-)
There are actually a whole world of datalevels (at least internally): 0,1, 1.5,2 1a, 1b etc ... Incredibly complex. 
I need to bring this question to the EBAS technicians. I think they actually could fill in the missing data levels, but it may be a large job, or alternatively you could possibly just skip the datalevel criteria for O3. Will respond to you as soon as I get a reply.

**Comment by SverreSolberg** [comment]
So, the message is: You should avoid any use of datalevels. Instead, for all species, you should select data based on project association = 'EMEP' and 'EMEP_preliminary'. The last part is very important since most of the 2019 data are presumably still 'EMEP_preliminary' ;-)

**Comment by jgliss** [comment]
> So, the message is: You should avoid any use of datalevels. Instead, for all species, you should select data based on project association = 'EMEP' and 'EMEP_preliminary'. The last part is very important since most of the 2019 data are presumably still 'EMEP_preliminary' ;-)

okay, but only for O3, right? For all other variables, I use datalevel=2, and no specific "project association"?

**Comment by SverreSolberg** [comment]
No, use this for all species. To be 100 % sure you should ask Wenche, I think.
